cogeo_mosaic/backends/dynamodb.py

```diff
@@ -66,20 +70,20 @@ def _create_table(self, billing_mode: str = "PAY_PER_REQUEST"):
                 KeySchema=key_schema,
                 BillingMode=billing_mode,
             )
-            logger.info("creating table")
 
             # If outside try/except block, could wait forever if unable to
             # create table
             self.table.wait_until_exists()
-        except boto3.client("dynamodb").exceptions.ResourceInUseException:
-            logger.warn("unable to create table, may already exist")
+        except self.client.exceptions.ResourceInUseException:
+            warnings.warn("Unable to create table, may already exist")
+            return
```

Comment:
You don't really need this `return`, but I guess that's more a stylistic decision